src/explicitode.jl

```diff
@@ -373,7 +369,7 @@ function taylorinteg{T<:Number}(f, q0::Array{T,1}, t0::T, tmax::T,
         end
     end
 
-    return view(tv,1:nsteps), view(xv,:,1:nsteps)' #for xv, first do view, then transpose (otherwise it crashes)
+    return view(tv,1:nsteps), view(transpose(xv),1:nsteps,:)
```

Comment:
I note the following:
```julia
julia> v = randn(1000, 1000);

julia> b = transpose( view(v, :, :) );

julia> c = view( transpose(v), :, : );

julia> sizeof(v)
8000000

julia> sizeof(b)
8000000

julia> sizeof(c)
24
```

So `c` is truly a view; I thus agree, it seems better to use `view(transpose(xv),1:nsteps,:)`.